pylinac/winston_lutz.py

```diff
@@ -366,6 +366,10 @@ def __init__(self, file):
         self.check_inversion()
         # Flip the image upside down... makes it more intuitive later on.
         self.flipud()
+        # The first pixel line on the left has a glitch on Elekta iview EPID
```

Comment:
Okay, that's fine. I just wanted to make sure this wasn't just your machine and no one else's. I'll merge this and add some tests and release a bugfix update. Thanks for your contribution!